### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,8 @@
+[tox]
+envlist = py37, flake8
+basepython = 37
```
Comment (brianhelba) [inline tox.ini]: `basepython` here is illegal: https://tox.readthedocs.io/en/latest/config.html

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+
 [testenv:flake8]
```
Comment (brianhelba) [inline tox.ini]: If you want to call this `lint`, go for it.

File: tox.ini [hunk 3]
```diff
@@ -10,8 +13,7 @@ deps =
 commands =
     flake8
 
-[testenv:pytest]
-;basepython = py37
+[testenv:py37]
```
Comment (brianhelba) [inline tox.ini]: Change to `[testenv]`

**Review by brianhelba — CHANGES_REQUESTED** [review]
See

**Comment by danlamanna** [comment]
I fixed thisk

**Comment by danlamanna** [comment]
I fixed this up a bit, but this definitely does not work on a clean checkout. All the tests fail because of the data files not being checked out. I tried adding a commands_pre in the tox setup, but it's very difficult to test because git lfs caches the objects and makes it hard to clean everything out.

**Review by brianhelba — APPROVED** [review]

**Pull request merged**